Sofa/Sofa/Networking/APIClient.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  APIClient.swift
+//  Sofa
+//
+//  Created by Akademija on 05.04.2025..
+//
+
+import Foundation
+
+struct APIClient {
```

Comment:
Kreni odmah gledati ApiClient tako da ćeš imati 20 endpointa... Pa vidi što će ti se u tom slučaju duplicirati i odmah generaliziraj :)